app/ui.R

```diff
@@ -1,55 +1,82 @@
+               
+              fluidRow(
+                column(4, 
+                       selectInput("smr_in", "SMR", choices = c("(All)", unique(smr_completeness$smr)))
                           ),
-                   column(3,
-                          selectInput("data_item_in", "Data Item", choices = c("(All)", unique(smr_completeness$data_item)))
+                column(4,
+                       selectInput("hb_in", "Health Board", choices = c("(All)", unique(smr_completeness$hb_name)))
                           ),
-                   column(3,
-                          selectInput("percentage_in", "Percentage Complete", choices = c("(All)","0% - 20%", "20% <", "50% <", "80% <", "100%" ))
-                   )
-                        ),
-               fluidRow(
-                 tableOutput("completeness_table")
-                      )
+                column(4,
+                       selectInput("data_item_in", "Data Item", choices = NULL)
+                          )
+                     ),
+              
+              fluidRow(
+                checkboxGroupInput("percentage_in", 
+                                   "Completeness percentage range",
+                                   choices = list("Above 60% complete" = 1,
+                                                  "Between 40% and 60% complete" = 2,
+                                                  "Below 40% complete" = 3),
+                                   selected = c(1,2,3),
+                                   inline = TRUE
+                                   )
+                    ),
+                      
+              
+              fluidRow(
+                #add icon colors
+                tags$style(".glyphicon-ok {color:#2BE532}
+                          .glyphicon-warning-sign {color:#FFC300}
+                          .glyphicon-flag {color:#C70039}"),
+                DT::dataTableOutput("completeness_table")
+                    )
                ),
+      
       tabPanel("Timeliness", "Panel three contents"),
+      
       tabPanel("Accuracy Scores from SMR Audits",
+               
                fluidRow(
                  column(3,
                         selectInput("SMRaudit", "SMR", choices = c("(All)", unique(smr_audit$audit)))
                  ),
                  column(3, selectInput("Year", "Year", choices = NULL)
                  ),
                  
-                 column(3, selectInput("Healthboard", "Health Board", choices = NULL)
+                 column(3, selectInput("Healthboard", "Health Board", choices = c("(All)",unique(smr_audit$healthboard)))
                  ),
                  
                  column(3, selectInput("DataItemName", "Data Item", choices = NULL)
                  )
                ),
                
                fluidRow(
-                 tableOutput("audit_data")
+                 
+                 DT::dataTableOutput("audit_data")
                )
       )
```

Comment:
Similar to above, the `fluidRow` component needs changes. There are also 2 'Health Board' options?